src/mmap.rs

```diff
@@ -1138,8 +1187,10 @@ mod tests {
         let f = TempFile::new().unwrap().into_file();
         f.set_len(0x400).unwrap();
 
-        let gm = GuestMemoryMmap::from_ranges(&[(GuestAddress(0x1000), 0x400)]).unwrap();
-        let gm_backed_by_file = GuestMemoryMmap::from_ranges_with_files(&[(
+        let gm =
+            GuestMemoryMmap::from_ranges(&[GuestMmapRange::new(GuestAddress(0x1000), 0x400, None)])
```

Comment:
```suggestion
GuestMemoryMmap::from_ranges(&[GuestMmapRange::new(GuestAddress(0x1000), 0x400, None)])
```

Reply:
Hm that's bizarre, I don't know/remember why I posted those (maybe github was showing me the wrong indentation?) In any case sorry for the noise @vireshk